pkg/tle/tle.go

```diff
@@ -50,12 +50,22 @@ func GenerateTransparencyLogEntry(anon models.LogEntryAnon) (*rekor_pb.Transpare
 		inclusionProofHashes[i] = hashBytes
 	}
 
-	b, err := base64.StdEncoding.DecodeString(anon.Body.(string))
-	if err != nil {
-		return nil, fmt.Errorf("base64 decoding body: %w", err)
+	// Different call paths may supply string or []byte. If string, it is base64 encoded.
```

Comment:
I wonder if we should instead require a type here. Not a strong preference though. @bobcallaway?

Reply:
what call paths send a `[]byte`? I think that's more likely a bug we need to fix

Reply:
Discussed offline. We will leave this PR as-is for now and will follow up on this.